doc/index.yml

```diff
@@ -42,6 +42,7 @@ sections:
     - thingpedia-api
     - installing-almond-cloud
     - voice-api
+    - google-assistant
```

Comment:
I think this fits more under the "Use Cases" section of the docs.

Reply:
Changed to Use Cases